src/util-internal.h

```diff
@@ -132,6 +132,9 @@ is_route_rule_present(const NetplanNetDefinition* netdef, const NetplanIPRule* r
 NETPLAN_INTERNAL gboolean //FIXME: avoid exporting private symbol
 is_string_in_array(GArray* array, const char* value);
 
+NETPLAN_INTERNAL gboolean
+is_auth_key_management_psk(const NetplanAuthenticationSettings* auth);
```

Comment:
todo: This shouldn't be exported as a public symbol. (Yes, the `NETPLAN_INTERNAL` name is confusing, such symbols actually are public, but mean to be used by "Netplan tools" – e.g. CLI – only. If exported they should start with a `_netplan_...` prefix, to mark them as "private").

Reply:
todo: Can we please drop the `NETPLAN_INTERNAL` flag here, I think it's not needed.